app/root.tsx

```diff
@@ -40,8 +40,8 @@ import {
 	DropdownMenuTrigger,
 } from './components/ui/dropdown-menu.tsx'
 import { Icon, href as iconsHref } from './components/ui/icon.tsx'
-import fontStylestylesheetUrl from './styles/font.css'
-import tailwindStylesheetUrl from './styles/tailwind.css'
+import fontStyleStyleSheetUrl from './styles/font.css'
```

Comment:
I checked with MDN, and they refer to Stylesheet as StyleSheet. It felt wrong looking at it.

https://developer.mozilla.org/en-US/docs/Web/API/StyleSheet

Reply:
Maybe it should be `fontStyleSheetUrl` 😅

Reply:
Yessir, I missed this with renaming haha.